.idea/misc.xml

```diff
@@ -6,4 +6,4 @@
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```

Comment:
Please  revert--this seems like an unintentional change (also, note that we always have EOF line endings on our files though Android Studio will not add them by default).

Reply:
Whoops, my bad - fixed!